pyartcd/pyartcd/pipelines/rebuild.py

```diff
@@ -207,7 +207,7 @@ async def _build_plashet_from_tags(self, name: str, directory_name: str, el_vers
         else:
             await exectools.cmd_assert_async(cmd, env=self._doozer_env_vars)
 
-        remote_url = constants.PLASHET_REMOTE_URL + f"/{major}.{minor}"
+        remote_url = constants.PLASHET_REMOTES[0]['url'] + f"/{major}.{minor}"
```

Comment:
I'm thinking something like PLASHET_REMOTE = {..} and PLASHET_REMOTE_NEW = {..} would help readability and make it clear which one is being used.

Reply:
also why isn't this using self.plashet_remote ?

Reply:
Updated by using `self.plashet_remote`